src/components/tune.ts

```diff
@@ -50,42 +55,78 @@ export class TuneComponent {
       configIndex: ConfigIndex.PROTOCOL,
       url: 'protocol',
       title: 'Communication protocol',
+      displayReversed: false,
+      editable: false,
+      format: (x) => `${x}`,
+      parse: (x) => Number(x),
       presets: [
-        {index: 0, name: 'Windows', desc:'', value:'XInput', leds:0b0001, blink:0b1000},
-        {index: 1, name: 'Linux', desc:'and Steam Deck', value:'XPad', leds:0b0001, blink:0b0100},
-        {index: 2, name: 'Generic', desc:'aka DirectInput', value:'HID', leds:0b0001, blink:0b0010},
+        {index: 0, name: 'Windows', desc:'',                value:'XInput', leds:0b0001, blink:0b1000},
+        {index: 1, name: 'Linux',   desc:'and Steam Deck',  value:'XPad',   leds:0b0001, blink:0b0100},
+        {index: 2, name: 'Generic', desc:'aka DirectInput', value:'HID',    leds:0b0001, blink:0b0010},
       ]
     },
     touch_sens: {
       configIndex: ConfigIndex.SENS_TOUCH,
       url: 'touch_sens',
       title: 'Touch sensitivity',
+      displayReversed: true,
+      editable: true,
+      format: (x) => `${x} μs`,
+      parse: (x) => {
+        const re = RegExp('^([0-9]{1,3}) μs$').exec(x)
+        if (re) {
+          let value = Number(re[1])
```

Comment:
Add comment about what happens to [0]
More importantly, that this returns what is in the bracket